### ISSUE TITLE
[low] Chrome + Jaws: Screen reader announce 'okay got it' twice. 

### ISSUE THREAD

**Issue by Stutikuls** [Expensify/App#8623]
## Action Performed:
1. Using JAWS + Chrome browser, open the URL new.expensify.com/r/90156971                                                                                    
2. Open settings, navigate to the security using Tab key and press Enter.                                                      
3. Go to the close account using Tab key and press Enter.   
4. New content will appear, give inputs to the field and go to close account using Tab key and press Enter.
5. Pop-up will appear, go to Okay got it button using tab key and listen to the screen reader announcement.

## Expected Result:
screen reader should announce 'okay got it' only once.

## Actual Result:
Screen reader announce 'okay got it' twice. 

HTML
`<div class="css-1dbjc4n r-1777fci" style="background-color: rgb(3, 212, 124); border-radius: 8px; border-width: 0px; height: 52px; opacity: 1; padding-right: 12px; padding-left: 12px;"><div dir="auto" class="css-901oao r-633pao r-lrvibr" style="color: rgb(255, 255, 255); font-family: GTAmericaExp-Regular; font-size: 15px; font-weight: 700; line-height: 20px; text-align: center;">Okay, Got it</div></div>`

**Other occurrences**
>Same  issue repro on PR #8940 -Screen reader announce password twice.
>Same  issue repro on PR #9347  -Screen reader announce 'Invite' and 'Remove' twice.

## Workaround:
Yes

## Area issue was found in:
Security

## Failed WCAG checkpoints
1.3.1

## User impact:
Screen reader users hear the announcements twice that is annoying and confusing.

## Suggested resolution:
Ensure the Number of slide announced only once by the screen reader.             
Refer to: 
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships

## Platform:

- Web

**Version Number:** v1.1.49-1
**Reproducible in staging?:** Yes
**Reproducible in production?:** Yes
**Issue reported by:** Stuti

https://user-images.githubusercontent.com/97600950/163131981-5b50fecd-c16e-4092-a28d-1f329f8f459a.mp4
